packages/sonarwhal/src/lib/utils/content-type.ts

```diff
@@ -250,6 +250,18 @@ const determineMediaTypeBasedOnFileExtension = (resource: string): string => {
     return getMediaTypeBasedOnFileExtension(fileExtension);
 };
 
+const determineMediaTypeBasedOnFileName = (resource: string) => {
+    const fileName = getFileName(resource);
+
+    if (!fileName) {
+        return null;
+    }
+
+    const configFileNameRegex = /^\.[a-z]+rc$/i;
+
+    return configFileNameRegex.test(fileName) ? 'text/json' : null;
```

Comment:
IMHO, most .<something>rc file are not JSON. We should do a `JSON.parse(content)` and try catch, and if it the content can be parsed as JSON, then it's JSON.

Reply:
And otherwise end text/plain or something like that